bluepyefe/reader.py

```diff
@@ -199,75 +248,43 @@ def nwb_reader_BBP(in_data):
     _check_metadata(
         in_data,
         nwb_reader_BBP.__name__,
-        ["filepath", "i_unit", "v_unit", "t_unit", "protocol_name"],
+        ["filepath", "protocol_name"],
     )
 
     filepath = in_data["filepath"]
-    r = h5py.File(filepath, "r")
+    content = h5py.File(filepath, "r")
```

Comment:
It'd be good to close the h5 file afterwards. Not calling `content.close()` method can cause corruption to the h5 file.
Another alternative is to open it inside a context manager. This also makes sure the file gets closed even if there's a runtime exception.
`with h5py.File(filepath, "r") as content:`

Reply:
Yeah, I'd go for the context manager.

Reply:
Done !